api/infrastructure/parallelcluster-api.yaml

```diff
@@ -59,6 +59,23 @@ Parameters:
       - true
       - false
 
+  EnableFSxS3Access:
+    Description: |
+      When set to true the ParallelCluster API can access, write to the S3 buckets specified in the Filed FsxS3Bucket, it is needed to import/export from/to S3 when creating an FSx filesystem.
+      WARNING - setting this to true grants to the Lambda function S3 Get*, List* and PutObject privileges on the buckets specified in FsxS3Buckets.
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+
+  FsxS3Buckets:
+    Description: |
+      Comma separated arn of the S3 buckets, to allow the lambda function to import/export from/to S3 when creating an FSx filesystem.
+      WARNING - The setting is used only when EnableFSxS3Access is set to true. (example arn:aws:s3:::<S3_BUCKET_1>,arn:aws:s3:::<S3_BUCKET_2>)
+    Type: String
+    AllowedPattern: ^($|arn:.*:s3:[a-z0-9\-]*:([0-9]{12})*:.+)
```

Comment:
Where did you get this regex?
Be careful that it matches strings such as "arn:aws:s3:::S3_BUCKET_1,   arn:aws:s3:::S3_BUCKET_2", (with blank spaces), that are not split as expected within the policy statement

Reply:
It's a custom regex. I'll test it against such variations of S3 arns.